core/src/test/java/io/fabric8/maven/core/handler/ReplicationControllerHandlerTest.java

```diff
@@ -111,7 +111,7 @@ public void replicationControllerHandlerTest() {
         assertFalse(replicationController.getSpec().getTemplate().getSpec().getVolumes().isEmpty());
         assertEquals("test",replicationController.getSpec().getTemplate().getSpec().
                 getVolumes().get(0).getName());
-        assertEquals("/test/path",replicationController.getSpec().getTemplate()
+        assertEquals("path",replicationController.getSpec().getTemplate()
```

Comment:
Are you sure that you want to change the assert only ?

I mean, in `before()` you set up the path to "/test/path", so I'm not sure why you would expect a relative directory now in the generated objects.

Reply:
Oops, I interchanged parameters here ;-) . Thanks for noticing :smile: 
https://github.com/fabric8io/fabric8-maven-plugin/blob/d9d74f21c2289d0c0f85ec0b4f37d65869553b58/core/src/main/java/io/fabric8/maven/core/config/VolumeType.java#L32